sql/chunk.sql

```diff
@@ -90,3 +90,9 @@ RETURNS BOOL AS '@MODULE_PATHNAME@', 'ts_chunk_freeze_chunk' LANGUAGE C VOLATILE
 CREATE OR REPLACE FUNCTION _timescaledb_internal.drop_chunk(
    chunk REGCLASS)
 RETURNS BOOL AS '@MODULE_PATHNAME@', 'ts_chunk_drop_single_chunk' LANGUAGE C VOLATILE;
+
+-- internal API used by OSM extension to attach a table as a chunk of the hypertable
+CREATE OR REPLACE FUNCTION _timescaledb_internal.attach_osm_table_chunk(
```

Comment:
I had a more generic name , attach_foreign_table_chunk, for my first pass.  I decided to give it a more specific name so that are no misconceptions - e.g. any foreign table can be attached to the hypertable.  For instance, this would not work for the multinode case. There are  constraints on the foreign table chunk that are specific to OSM.